react-charts/area-chart/step2.md

```diff
@@ -1,24 +1,51 @@
-We'll begin by adding a CSS selector to define the chart size...
-
-- Katacoda is setting up a new React application for you. You'll be ready to code once the server starts and you can see "Welcome to PatternFly" on the lower pane.
+Now you will create a simple area chart.
 
 ## Task
 
-1) Navigate to the `src` folder and open `src/app.css`{{open}}
-
-2) Next we'll add some CSS to define the overall chart size
-
-Copy the following code into the app.css file, replacing all of the content there:
-
-<pre class="file" data-filename="src/app.css" data-target="replace">
-.chart-container {
-  height: 250px;
-  width: 600px;
+1) <strong>Navigate to the `src` folder and open `src/App.js`{{open}}</strong>
+
+2) Next, you will set up the general structure of the chart and the necessary import statements. <strong>Copy the following code into the App.js file, replacing all of the content there:</strong>
```

Comment:
Let's just put, "Set up the general..." Because these steps are numbered, we don't need the "Next" included. And we can stick to present tense by cutting the "you will."